var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -328,7 +327,8 @@ def configure_args(self):
         if sys.platform == 'darwin':
             options.extend([
                 '--with-native-system-header-dir=/usr/include',
-                '--with-sysroot={0}'.format(macos_sdk_path())
+                '--with-sysroot={0}'.format(macos_sdk_path()),
+                '--with-libiconv-prefix={0}'.format(spec['libiconv'].prefix),
```

Comment:
Is the trailing comma here on purpose?

Reply:
Dang it!  Cut & paste error.  Thanks for spotting that issue. Update is incoming...